sql/core/src/test/resources/sql-tests/results/describe-table-column.sql.out

```diff
@@ -0,0 +1,133 @@
+-- Automatically generated by SQLQueryTestSuite
+-- Number of queries: 15
+
+
+-- !query 0
+CREATE TEMPORARY VIEW desc_col_temp_table (key int COMMENT 'column_comment') USING PARQUET
+-- !query 0 schema
+struct<>
+-- !query 0 output
+
+
+
+-- !query 1
+DESC desc_col_temp_table key
+-- !query 1 schema
+struct<col_name:string,data_type:string,comment:string>
+-- !query 1 output
+col_name 	data_type 	comment        	
+key      	int       	column_comment
+
+
+-- !query 2
+DESC EXTENDED desc_col_temp_table key
```

Comment:
Since we already treat `DESC EXTENDED` and `DESC FORMATTED` identical for the tables, let us make DESC EXTENDED and FORMATTED the same for the column too.